### ISSUE TITLE
WCAG - Understanding SC 4.1.2 - Ensure Buttons have discernable text

### ISSUE THREAD

**Issue by JonCole** [grafana/grafana#106184]
**Steps to reproduce**:
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights. 

1. TAB to "Add visualization" button and hit ENTER key. 
2. "Select data source" popup will appear. 
3. Select "Azure monitor" default data source 
4. Run Automated check from Accessibility insights for web and observe the issue. 

**Actual Result**:

<button class="css-4slnse-button" type="button" data-testid="data-testid Options group Panel options toggle" aria-expanded="true" aria-disabled="false"> 

**Expected Result**

Fix any of the following: 
  1.   Element does not have inner text that is visible to screen readers 
  2.   aria-label attribute does not exist or is empty 
  3.   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty 
  4.   Element has no title attribute 
  5.   Element does not have an implicit (wrapped) <label> 
  6.   Element does not have an explicit <label> 
  7.   Element's default semantics were not overridden with role="none" or role="presentation" 

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Environment**:
- Grafana version: 11.5.1
- Data source type & version:
- User OS & Browser: Windows 11 Pro, Microsoft Edge version 136.0.0.0 
- Others:

**Comment by LinglingTong** [comment]
Severity: Sev2
